src/spring-cloud/azext_spring_cloud/tests/latest/test_asc_app_insights_scenario.py

```diff
@@ -0,0 +1,369 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import time
+
+from azure.cli.testsdk import (ScenarioTest, record_only)
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+@record_only()
```

Comment:
May I ask why this test is `@record_only()`?

Reply:
The request and response has been recorded in yaml files under "/spring-cloud/azext_spring_cloud/tests/latest/recordings". And the `recording_only` will indicate it to run against them. This is for one.

For the second reason, these tests will interact with backend service. The purpose is to test the output of CLI, so don't need to really call to back-end.

reference for `live_only` and `record_only`:
```
def live_only():
    return unittest.skipUnless(
        os.environ.get(ENV_LIVE_TEST, False),
        'This is a live only test. A live test will bypass all vcrpy components.')


def record_only():
    return unittest.skipUnless(
        not os.environ.get(ENV_LIVE_TEST, False),
        'This test is excluded from being run live. To force a recording, please remove the recording file.')
```

Reply:
I know the function of `@record_only()`. What I want to ask is that why this test can't live run again?